src/mscorlib/src/System/String.Manipulation.cs

```diff
@@ -541,6 +541,63 @@ public String Insert(int startIndex, String value)
             }
             return result;
         }
+
+        public static string Join(char separator, params string[] value)
+        {
+            if (value == null)
+            {
+                throw new ArgumentNullException(nameof(value));
+            }
+
+            return Join(separator, value, 0, value.Length);
+        }
+
+        public static string Join(char separator, params object[] values)
```

Comment:
When I added the new `Split` overloads, I made them `[ComVisible(false)]`. I think we should do the same for these new `Join` overloads.